public/app/plugins/datasource/prometheus/plugin.json

```diff
@@ -105,5 +105,6 @@
         "url": "https://prometheus.io/"
       }
     ]
-  }
+  },
+  "apiVersion": "v0alpha3"
```

Comment:
^^^ should be removed before commit!  

But note, that this will make the server run on the configured apiVersion 🎉 

http://localhost:3000/apis/prometheus.datasource.grafana.app/v0alpha3/namespaces/default/connections/gdev-prometheus